subprojects/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/tasks/AbstractPluginValidationIntegrationSpec.groovy

```diff
@@ -816,7 +840,9 @@ abstract class AbstractPluginValidationIntegrationSpec extends AbstractIntegrati
             import org.gradle.api.model.*;
             import org.gradle.api.tasks.*;
             import org.gradle.api.provider.*;
+            import org.gradle.work.*;
 
+            @DisableCachingByDefault(because = "my task")
```

Comment:
Should we use something like `test task` instead?

Reply:
Done. I could also remove those annotations, given that we don't validate them in non-strict mode. WDYT?